src/cljs/ataru/hakija/application_handlers.cljs

```diff
@@ -79,13 +79,7 @@
 (reg-event-fx
   :application/submit
   (fn [{:keys [db]} _]
-    (send-application db :submit)
```

Comment:
`send-application` has no side effects, so it's just dead code here (also there's no HTTP method named `submit`)... I removed the extracted form of the returned map and replaced it with the send-application (with `post` method) which seems to be functionally identical.